examples/nlp/language_modeling/constants.py

```diff
@@ -132,4 +133,192 @@
+        "query_start": "",
+        "response": "The final answer is",
+        "tokens_to_generate": 32,
+        "subset": "LongBench/passage_count.jsonl",
+        "chat_instruction": "",
+        "template": "There are some paragraphs below sourced from Wikipedia. Some of them may be duplicates. Please carefully read these paragraphs and determine how many unique paragraphs there are after removing duplicates. In other words, how many non-repeating paragraphs are there in total?\n\n{context}\n\nPlease enter the final count of unique paragraphs after removing duplicates. The output format should only contain the number, such as 1, 2, 3, and so on.\n\nThe final answer is: ",
+    },
+    "passage_retrieval_en_lb": {
+        "context": "Please determine which paragraph the abstract is from.\n\n",
+        "query_start": "The following is an abstract.",
+        "response": "The answer is:",
+        "tokens_to_generate": 32,
+        "subset": "LongBench/passage_retrieval_en.jsonl",
+        "chat_instruction": "",
+        "template": "Here are 30 paragraphs from Wikipedia, along with an abstract. Please determine which paragraph the abstract is from.\n\n{context}\n\nThe following is an abstract.\n\n{input}\n\nPlease enter the number of the paragraph that the abstract is from. The answer format must be like \"Paragraph 1\", \"Paragraph 2\", etc.\n\nThe answer is: ",
+    },
+    "passage_retrieval_zh_lb": {
+        "context": "请确定给定的摘要出自哪一段。\n\n",
+        "query_start": "\u6458\u8981\n\n",
+        "response": "答案是:",
+        "tokens_to_generate": 32,
+        "subset": "LongBench/passage_retrieval_zh.jsonl",
+        "chat_instruction": "",
+        "template": "\u4ee5\u4e0b\u662f\u82e5\u5e72\u6bb5\u843d\u6587\u5b57\uff0c\u4ee5\u53ca\u5176\u4e2d\u4e00\u4e2a\u6bb5\u843d\u7684\u6458\u8981\u3002\u8bf7\u786e\u5b9a\u7ed9\u5b9a\u7684\u6458\u8981\u51fa\u81ea\u54ea\u4e00\u6bb5\u3002\n\n{context}\n\n\u4e0b\u9762\u662f\u4e00\u4e2a\u6458\u8981\n\n{input}\n\n\u8bf7\u8f93\u5165\u6458\u8981\u6240\u5c5e\u6bb5\u843d\u7684\u7f16\u53f7\u3002\u7b54\u6848\u683c\u5f0f\u5fc5\u987b\u662f\"\u6bb5\u843d1\"\uff0c\"\u6bb5\u843d2\"\u7b49\u683c\u5f0f\n\n\u7b54\u6848\u662f\uff1a",
+    },
+    "lcc_lb": {
+        "context": "Please complete the code given below. \n",
+        "query_start": "",
+        "response": "Next line of code:",
+        "tokens_to_generate": 64,
+        "subset": "LongBench/lcc.jsonl",
+        "chat_instruction": "",
+        "template": "Please complete the code given below. \n{context}Next line of code:\n",
+    },
+    "repobench-p_lb": {
+        "context": "Please complete the code given below. \n",
+        "query_start": "",
+        "response": "Next line of code:",
+        "tokens_to_generate": 64,
+        "subset": "LongBench/repobench-p.jsonl",
+        "chat_instruction": "",
+        "template": "Please complete the code given below. \n{context}{input}Next line of code:\n",
```

Comment:
Just confirming: Should we have a "space" or "\n" between `{context}` and `{input}`? Because I see that the other tasks have some separation between the two.

Reply:
Got it. Thanks.